CONTRIBUTING.md

```diff
@@ -19,19 +19,22 @@ branch structure is as following :
 **A pull request must be created and approved before merging with `develop`!**
 
 We use a **rebase** strategy when merging to `develop`. When a feature has been finished in
-development, the feature branch must be rebased with `develop` before merging in order to avoid
+development, the feature branch must be rebased with `develop` before creating the pull request in order to avoid
 merge commits.
 
-A rebase merge to `develop` can be done in command line like this:
+A rebase merge from `develop` to the feature branch can be done in command line like this:
 
 ```bash
 git fetch
 git checkout feature/author/feature-name # checkout the feature branch
 git rebase origin/develop # rebase with remote develop branch
 # resolve any conflicts
-git checkout develop
-git merge feature/author/feature-name
+# add or stage your changes
+git add .
```

Comment:
Adding all files can be unintended (I learned my lesson today 😇 ). I suggest to just have the comment about adding/staging your changes